### ISSUE TITLE
regression in `path-params` rule, complains about missing path params that are valid

### ISSUE THREAD

**Issue by TristanSpeakEasy** [daveshanley/vacuum#390]
I am seeing these errors showing up in the latest vacuum release

```
"validation error: [line 35] path-params - `POST` must define parameter `org` as expected by path `/test2/org/{org}/team/{team}` | $.paths['/test2/org/{org}/team/{team}'].post",
"validation error: [line 35] path-params - `POST` must define parameter `team` as expected by path `/test2/org/{org}/team/{team}` | $.paths['/test2/org/{org}/team/{team}'].post",
```

that are new regressions, the attached specs show the issue and are definitely valid.

[references.zip](https://github.com/daveshanley/vacuum/files/13566790/references.zip)

**Comment by TristanSpeakEasy** [comment]
@daveshanley any luck figuring out this one?

**Comment by daveshanley** [comment]
This has been resolved in `v0.9.0` It was fixed in `libopenapi` and a test was added to ensure it would not return.

https://github.com/pb33f/libopenapi/blob/main/index/spec_index_test.go#L1618
